ci-operator/config/open-cluster-management/observability-e2e-test/open-cluster-management-observability-e2e-test-main.yaml

```diff
@@ -48,6 +48,8 @@ tests:
       env:
       - default: oc
         name: KUBECTL
+      - default: ${IMAGE_FORMAT//\/stable:\$\{component\}//pipeline:observability-e2e-test}
+        name: COMPONENT_IMAGE_NAME
```

Comment:
Do we need this? I found the `observability-e2e-test` found in https://github.com/open-cluster-management/observability-e2e-test/blob/main/cicd-scripts/setup-e2e-tests.sh#L80